### PR TITLE
[MIG] [11.0] Migration of 'quality_control_mrp'

### PR DESCRIPTION
**Pull request by NachoAlesLopez**
@guadaltech

### CODE REVIEW

File: quality_control_mrp/models/qc_inspection.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            if inspection.object_id:
+                if inspection.object_id._name == 'stock.move':
+                    inspection.production_id = \
+                        inspection.object_id.production_id
+                elif inspection.object_id._name == 'mrp.production':
+                    inspection.production_id = inspection.object_id
+
+    @api.multi
+    @api.depends('object_id')
+    def _get_product(self):
```
Comment (mariaasunciondaza) [inline quality_control_mrp/models/qc_inspection.py]:
```suggestion
    def _compute_product_id(self):
```

**Review by mariaasunciondaza — CHANGES_REQUESTED** [review]
Please review it. Thanks!

File: quality_control_mrp/models/qc_inspection.py (hunk 1 above)
Comment (NachoAlesLopez) [reply quality_control_mrp/models/qc_inspection.py]: Thanks for the review! Should be fixed now.

**Review by mariaasunciondaza — APPROVED** [review]
LGTM. Thanks!

**Review by ps-tubtim — APPROVED** [review]
LGTM.

**Pull request merged**